File: abc061/C.rb
# https://atcoder.jp/contests/abc061/tasks/abc061_c


N, K = gets.split.map(&:to_i)
as = Array.new(N)
bs = Array.new(N)
N.times do |i|
  as[i], bs[i] = gets.split.map(&:to_i)
end

puts ans

File: abc061/C.cxx
// https://atcoder.jp/contests/abc061/tasks/abc061_c

#include <cstdio>

using namespace std;

#define REP(i,n)   for(int i=0; i<(int)(n); i++)
#define FOR(i,b,e) for(int i=(b); i<=(int)(e); i++)

const int N_MAX = 1e5;
const int A_I_MAX = 1e5;
const int B_I_MAX = 1e5;

int N, K;
int a[N_MAX];
int b[N_MAX];

void solve() {
  int ans = 0;
  printf("%d\n", ans);
}

void input() {
  scanf("%d%d", &N, &K);
  REP(i, N) scanf("%d%d", a + i, b + i);
}

int main() {
  input();
  solve();
  return 0;
}
